config/locales/details.pdf.yml

```diff
@@ -0,0 +1,56 @@
+en:
```

Comment:
Is there any way to reduce the duplication here? e.g. Jesus' suggestion of a fallback if lookup in the pdf namespace fails